src/main/java/hu/oe/nik/szfmv/automatedcar/model/World.java

```diff
@@ -4,15 +4,39 @@
 import java.util.List;
 
 import hu.oe.nik.szfmv.automatedcar.model.interfaces.IWorld;
+import hu.oe.nik.szfmv.automatedcar.xml.converter.IntegerConverter;
+
+import javax.xml.bind.annotation.*;
```

Comment:
*-al ne importáljunk, ha lehet, ha IntelliJ-d van tudok segíteni beállítani ezt

Reply:
Köszi, de felteszek majd egy frissebb NetBeanst, akkor már kényelmesen fogok tudni kódolni és nem kerülnek be majd ilyenek.  😉